test_setup_system.py

```
@@ -50,159 +56,163 @@ def create_test_environment(self, name: str, scenario: str) -> Path:
-            
+
         except subprocess.TimeoutExpired:
             console.print(f"⏰ {name} timed out")
-            return {
-                "name": name,
-                "scenario": scenario,
-                "success": False,
-                "error": "Timeout"
-            }
+            return {"name": name, "scenario": scenario, "success": False, "error": "Timeout"}
         except Exception as e:
```

Ruff (0.15.6)

[warning] Do not catch blind exception: `Exception`

(BLE001)